src/main/java/seedu/address/model/name/CommonName.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.name;
+
+public class CommonName {
```

Comment:
And we could call this class just Name?

Reply:
i don't think so , because it will be same to the name under booking